presentation/routes/customer.ts

```diff
@@ -1,9 +1,8 @@
 import { Router } from 'express';
 import UserController from '../controllers/UserController';
-import customerValidators from '../validators/customer-validators';
 
 const router = Router();
-router.post('/', customerValidators, UserController.createCustomer);
+router.post('/', UserController.createCustomer);
```

Comment:
poderia manter o userSchema aqui nas rotas, dessa forma, se ele não passar na validação no Joi ele nem vai para o controller, que acha?

Reply:
Você se refere a usar o Joi/userSchema como um middleware antes do UserController? Pensei que ele devesse ser parte do Service, mas se assim for melhor posso fazer essa mudança.

Reply:
Resolvido com o commit [6707396](https://github.com/Neo-WRN/typescript-challenge-user/pull/5/commits/6707396cdacbd79e5634a72fad8144040dc66c31).